tests/test-upgrade.py

```diff
@@ -96,6 +97,15 @@ def test_upgrade(self):
         except:
             print('Will not test registry')
 
+
+        try:
+            enable = microk8s_enable("reg")
```

Comment:
Can you please add a `timeout_insec=30` when enabling the add-on. If we do not set a timeout enabling tries for too long and tests on travis timeout.